lib/generators/hyrax/work_resource/templates/indexer_spec.rb.erb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Generated via
+#  `rails generate hyrax:work_resource <%= class_name %>`
+require 'rails_helper'
+require 'hyrax/specs/shared_specs/indexers'
+
+RSpec.describe <%= class_name %>Indexer do
+  let(:indexer_class) do
```

Comment:
doesn't this test the wrong class?

the `describe` says this tests `[GeneratedModel]Indexer`, is there a reason we'd want to test a generic `Hyrax::ResourceIndexer` instead?

Reply:
That was likely a copy/paste error.  I'll take a look.

Reply:
@no-reply Updated to correctly use the generated index class.